CMSgov/mint-app
# [EASI-3918] Notification Page 🎉

# EASI-3918


## Changes and Description

- Added new Notification routes
- Implemented Info banner when there are no notifications
- Notifications sorted in the order of most recent first
- Notification read states
- Link will update mutation to mark as read
- User has option to "Mark all as read"
- Show pagination after 10 items
- Add Translation mappings
- Add unit tests
- Add cypress test




## How to test this change

1. Navigate to `/notifications/`
2. Verify that info banner is there
3. Create  `TWO` discussions and tag self
4. Verify that Notification on the menu bar has a red dot (notifying that there is an update)
5.  Go back  to `/notifications/`
6. Verify that new notifications are showing
7. Hitting `Mark all as read` will remove the red dot and mark all them read
8. Hitting `View Discussion` will only mark that individual discussion
9. cypress and unit tests are not failing
10. try to break it
11. Test the Feature Flag in LD. 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
